SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -613,10 +613,10 @@ \subsection{Naming Files}
 \subsubsection{Naming Tests}
 \label{subsubsec:NamingTests}
 
-Following the \code{TestDriver} philosophy, test files must be named with
-the same name used to name the \code{main} method contained in the test file
-(\code{.cxx}). This name should generally be indicative of the class tested,
-e.g.
+Following the \code{TestDriver} philosophy, Section~\ref{sec:Tests}, test
```

Comment:
AFAIK, in the ITK SW Guide it is customary to add the **pageref** as well e.g. `Section~\ref{sec:Tests} on page~\pageref{sec:Tests}` for the sake of completion

Reply:
Adding.